.lintstagedrc

```diff
@@ -1,5 +1,6 @@
 {
   "*.js": [
+    "git add .",
```

Comment:
I'm also a bit dubious. Did this cause the .gitignore merge errors to be committed? If anything, perhaps it should only add the modified .js files.

 https://github.com/okonet/lint-staged#examples says:

> Please note, that it doesn’t work well with committing hunks (git add -p).